target/linux/ramips/dts/mt7620a_engenius_esr600.dts

```diff
@@ -155,7 +155,7 @@
 				0x7c 0x0000007e /* PORT0 STATUS */
 				0x0c 0x05600000 /* PORT6 PAD MODE CTRL */
 				0x94 0x0000007e /* PORT6 STATUS */
-				>;
+			>;
```

Comment:
That's unrelated, and I think we prefer the version before the change?

Reply:
I believe this change is correct formatting, but if you want I can remove it because it's unrelated

Reply:
Well, what is correct? I don't really want to waste time on designing a grep for this, but if you do ...

Reply:
ok let me be direct:

this is a correct change for formatting because just like other nodes, the lines inside the node block are indented with respect to the start of the node, and the characters that end the node block are not indented. (and for that matter, we do the same thing in shell scripts and in C programs)

Reply:
I understood that, but still, formatting is only convention. If most of the others do it differently in DTS, being "correct" is not really valuable ...
Anyway, it's unrelated here, so let's leave the discussion for some boring hour in the future ...